genie-core/build.gradle

```diff
@@ -16,12 +16,18 @@ dependencies {
     // Commons Libs
     compile("commons-httpclient:commons-httpclient")
     compile("commons-io:commons-io")
+    compile("org.apache.commons:commons-exec:${commons_exec_version}")
 
     // Spring Libs
     compile("org.springframework.boot:spring-boot-starter-aop")
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
     compile("org.springframework.boot:spring-boot-starter-mail")
     compile("org.springframework.retry:spring-retry")
+    compile("org.springframework.cloud:spring-cloud-aws-context:1.0.4.RELEASE")
```

Comment:
Move the version of these libraries to gradle.properties and reference.

does the java sdk not come as a transitive dependency of the spring cloud AWS dependency?